internal/cache.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2021 VMware, Inc.
```

Comment:
I'd be more specific on the naming of this file. It is a cache of what? I'd call it intermediateBundleCache or smth like that.

Reply:
Go source code filenames are never camel case and very rarely multi-word, I would avoid us being weird and non idiomatic there. Also seems snake case is preferred when using multiwords, which maps what is used for tests and example conventions.

Maybe I could call it `layer_cache.go` as it is for layer caching and does not even know or care about bundles itself.

Reply:
renamed as `layer_cache.go`